### DISCUSSION TITLE
Can't install PHP via Homebrew on MacOS 10.13.6

### DISCUSSION THREAD

**Discussion by timotif** [Homebrew/discussions#3356] [Everyday usage] [answered]
I have a hackintosh with High Sierra and the installation of php fails when it tries to upgrade Apache.
```
(base):~ $ brew install php
Warning: You are using macOS 10.13.
We (and Apple) do not provide support for this old version.
You will encounter build failures with some formulae.
Please create pull requests instead of asking for help on Homebrew's GitHub,
Twitter or any other official channels. You are responsible for resolving
any issues you experience while you are running this
old version.

==> Downloading https://dlcdn.apache.org/httpd/httpd-2.4.53.tar.bz2

curl: (22) The requested URL returned error: 404
Trying a mirror...
==> Downloading https://downloads.apache.org/httpd/httpd-2.4.53.tar.bz2

curl: (22) The requested URL returned error: 404 Not Found
Error: php: Failed to download resource "httpd"
Download failed: https://downloads.apache.org/httpd/httpd-2.4.53.tar.bz2
```

The version it's looking for is not there: 2.4.54 is the one available... so I thought homebrew might be outdated but it tells me it's up to date.
```
(base):~ $ brew -v
Homebrew 3.5.1-48-g4484068
Homebrew/homebrew-core (git revision 0712e880ba1; last commit 2022-06-12)
Homebrew/homebrew-cask (git revision 13eff273c6; last commit 2022-06-12)
```

Can I force around the installation? I don't need the latest php...

Thanks for your help!

**Comment by gromgit** [comment, marked as answer]
`httpd` is now at v2.4.54, though I'm not sure why the Apache folks removed all older versions from distribution. A version bump is [already in progress](https://github.com/Homebrew/homebrew-core/pull/103236), please be patient.

**Reply by timotif** [reply]
Thanks! I could go around it by downloading this https://archive.apache.org/dist/httpd/httpd-2.4.53.tar.bz2 which is the version it expects and placing it in the cache folder as it suggests here https://docs.brew.sh/Tips-N'-Tricks in the Pre-download section.

**Reply by gromgit** [reply]
Or since the `httpd` version bump is now completed, you could simply `brew update` and run the install again.

**Reply by timotif** [reply]
Thanks! It was good to learn the cache technique nevertheless :-) But I did upgrade to 2.4.54 now!
